README.md

```diff
@@ -53,6 +53,7 @@ The following developer guides cover specific topics for a technical audience.
 - [Meeting events](https://aws.github.io/amazon-chime-sdk-js/modules/meetingevents.html)
 - [Integrating Amazon Voice Focus into your application](https://aws.github.io/amazon-chime-sdk-js/modules/amazonvoice_focus.html)
 - [Adding frame-by-frame processing to an outgoing video stream](https://aws.github.io/amazon-chime-sdk-js/modules/videoprocessor.html)
+- [Adding priority based video downlink policy](https://aws.github.io/amazon-chime-sdk-js/modules/prioritybased_downlink_policy.html)
```

Comment:
I think this is sophisticated enough that we should start with what it solves: something like "Adapting video to limited bandwidth using a priority-based video downlink policy" — thoughts?